lib/public/UrlChecker.js

```diff
@@ -115,4 +122,33 @@ UrlChecker.prototype.resume = function()
 
 
 
+UrlChecker.prototype.clearCache = function()
+{
+	this.cache = {};
+};
```

Comment:
`HtmlChecker` and `HtmlUrlChecker` should have this method as well, calling this one. Example: https://github.com/stevenvachon/broken-link-checker/blob/master/lib/public/HtmlChecker.js#L97-L100